src/ray/util/logging.h

```diff
@@ -103,6 +103,8 @@ class RayLog : public RayLogBase {
   void *logging_provider_;
   /// True if log messages should be logged and false if they should be ignored.
   bool is_enabled_;
+  /// True if log messages should be stored in log file.
+  static bool is_stored_;
```

Comment:
This variable name seems a little bit unclear. Maybe just save `log_dir_` as a field here?

Reply:
Thanks!